scripts/config_osm_data.py

```diff
@@ -484,3 +484,241 @@
     "SN": "SNGM",  # senegal -> senegal-gambia
     "GM": "SNGM",  # gambia -> senegal-gambia
 }
+
+# Geofabrik and iso norm deviate for the following countries
+
+# Western Sahara -> Morocco ("EH": "MA")
+# Senegal -> Senegal-Gambia ("SN": "SNGM")
+# Gambia -> Senegal-Gambia  ("GM": "SNGM")
+# Hong Kong  -> China
+# Macao  -> China
+# Cyprus and Georgia -> European domain
+# Russia -> a separate domain
+
+# Data for some islands seem to be merged with some other areas data
+# That is the case for
+# "FO": "faroe islands"
+# "NF": "norfolk island",
+# "PF": "french-polynesia"
+# "GU": "guam"
+
+# "latin_america" -> "south-america"
+
+world_geofk = {
+    "africa": {
+        "DZ": "algeria",
+        "AO": "angola",
+        "BJ": "benin",
+        "BW": "botswana",
+        "BF": "burkina-faso",
+        "BI": "burundi",
+        "CM": "cameroon",
+        # canary-islands, # Island
+        # "CV": "cape-verde", # Island
+        "CF": "central-african-republic",
+        "TD": "chad",
+        # "KM": "comores", # Island
+        "CG": "congo-brazzaville",
+        "CD": "congo-democratic-republic",
+        "DJ": "djibouti",
+        "EG": "egypt",
+        "GQ": "equatorial-guinea",
+        "ER": "eritrea",
+        "ET": "ethiopia",
+        "GA": "gabon",
+        "GH": "ghana",
+        "GW": "guinea-bissau",  # No Data
+        "GN": "guinea",
+        "CI": "ivory-coast",
+        "KE": "kenya",
+        "LS": "lesotho",
+        "LR": "liberia",
+        "LY": "libya",
+        "MG": "madagascar",
+        "MW": "malawi",
+        "ML": "mali",
+        "MR": "mauritania",
+        # "MU": "mauritius", # Island
+        "MA": "morocco",
+        "MZ": "mozambique",
+        "NA": "namibia",
+        "NE": "niger",
+        "NG": "nigeria",
+        "RW": "rwanda",
+        # saint-helena-ascension-and-tristan-da-cunha # Islands
+        # "ST": "sao-tome-and-principe", # Island
+        "SNGM": "senegal-and-gambia",  # Geofk shortcurt
+        # "SC": "seychelles", # Island
+        "SL": "sierra-leone",
+        "SO": "somalia",  # No Data
+        # south-africa-and-lesotho
+        "ZA": "south-africa",
+        "SS": "south-sudan",
+        "SD": "sudan",
+        "SZ": "swaziland",
+        "TZ": "tanzania",
+        "TG": "togo",
+        "TN": "tunisia",
+        "UG": "uganda",
+        "ZM": "zambia",
+        "ZW": "zimbabwe",
+    },
+    "asia": {
+        "AF": "afghanistan",
+        "AM": "armenia",
+        "AZ": "azerbaijan",
+        "BD": "bangladesh",
+        "BT": "bhutan",
+        # 'IO': 'british indian ocean territory', # Island
+        "KH": "cambodia",
+        "CN": "china",
+        # 'CX': 'christmas island', # Island
+        # 'CC': 'cocos (keeling) islands', # Island
+        "GCC":
```

Comment:
Thank you, @davide-f! Actually, it's a good idea to check completeness of the whole geofabric-related dictionary. I'll add a test for it. As far, the `world_geofk` dictionary is tested only to ensure that the Geofabric urls are reachable.

By the way, could you please advice if there is any proper place in the project to store such testing scripts? The scripts folder doesn't look a right place for such purely supplementary kind of things...

Reply:
Sorry, GCC stands for "gulf cooperaton council" at the following line; I suggest to format it similarly to the other and put them in line.
Moreover, note that any geofk shortcut or agglomeration or alike may need to have the proper list of codes, each for iso coding in the iso_to_geofk dictionary.
Amazing job!

Regarding the script, instead of the script, may you use a notebook instead with some comments and description?
It could be put in the notebook folder and be useful for other people for better understanding geofk

Reply:
@davide-f Absolutely agree regarding geofk shortcuts. And love the idea of an explaining notebook. Thank you!